_includes/about.md

```diff
@@ -1,4 +1,4 @@
-Async meets twice a month in Brighton & Hove, UK, to learn, discuss and play
+Async meets once a month in Brighton & Hove, UK, to learn, discuss and play
```

Comment:
Can you split this PR up a bit please? Reviewers only have a limited concentration span